src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2427,6 +2548,43 @@ def aks_rotate_certs(cmd, client, resource_group_name, name, no_wait=True):
     return sdk_no_wait(no_wait, client.rotate_cluster_certificates, resource_group_name, name)
 
 
+def fuzzy_match(query, arr):
+    def similarWord(a, b):
```

Comment:
This is not pythonic, please use `similar_word` instead. And don't forget following variable name such as aLen, bLen, foundDifference, etc.

Reply:
Got it.